app/src/main/java/org/mozilla/focus/telemetry/TelemetrySettingsProvider.java

```diff
@@ -61,6 +61,9 @@ public boolean containsKey(String key) {
                 // However we still want to report the current search engine of the user.
                 // Therefore we inject this value at runtime.
                 value = SearchEngineManager.getInstance().getDefaultSearchEngine(context).getName();
+            } else if (SearchEngineManager.getInstance().isCustomSearchEngine((String) value, context)) {
+                // We only want to collect "custom" or "bundled", not the search engine itself.
```

Comment:
Why is "custom" or "bundled" relevant if we just specify "custom"? I'd suggest removing "bundled", if it's relevant.

Also, I'd include our motivation, i.e. explain that we don't want to collect personal info, like a custom search engine, so we replace it with a static value.